app/pocket/rpc/v1/openapi.yaml

```diff
@@ -0,0 +1,98 @@
+openapi: 3.0.3
```

Comment:
Just wanted to share: https://editor.swagger.io/?url=https://raw.githubusercontent.com/deblasis/pocket/issue/utility_local_proof_of_stake_cli_CLI/app/pocket/rpc/v1/openapi.yaml

We can add it to the docs in the next PR